en/core-libraries/caching.rst

```diff
@@ -382,6 +382,16 @@ save multiple network connections when using Memcached::
     ]);
     // $result will contain
     ['article-first-post' => true, 'article-first-post-comments' => true]
+    
+Deleting Group From Cache
+------------------------------
+
+.. php:staticmethod:: clearGroup($group, $config = 'default')
```

Comment:
There [is an existing section for this method](https://book.cakephp.org/3.0/en/core-libraries/caching.html#Cake\Cache\Cache::clearGroup).

Reply:
ohh my fault! sorry